### DISCUSSION TITLE
Project status

### DISCUSSION THREAD

**Discussion by LeviSnoot** [LeviSnoot/FNFest-Status#21] [Announcements]
It's been a bit, so I thought I'd provide a bit of an update.

I'm currently prioritizing other projects, so the rust rewrite is on hiatus for now, but if issues or PRs crop up, I will take a look at them, don't worry. With Festival Season 6 displaying the currently playing song and artist throughout the whole song, I haven't felt a need to further development of the stream widget itself anyway. Making further changes right before a new chapter also seems like a bad idea in case it breaks something. We'll see what happens.

TL;DR: This project is still on my radar, but not high on my list of priorities at the moment.
